spec/fake_braintree/transaction_spec.rb

```diff
@@ -11,6 +11,16 @@
       expect(result.transaction.type).to eq 'sale'
     end
 
+    it "sets the creation time" do
+      transaction = Braintree::Transaction.sale(
+        payment_method_token: cc_token,
+        amount: 10.00
+      ).transaction
+
+      creation_time = Time.parse(transaction.created_at)
+      expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.